ngraph/frontend/paddlepaddle/src/op/stack.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <ngraph/opsets/opset8.hpp>
+#include <node_context.hpp>
+
+namespace ngraph {
+namespace frontend {
+namespace pdpd {
+namespace op {
+NamedOutputs stack(const NodeContext& node) {
+    auto datas = node.get_ng_inputs("X");
+    auto axis = node.get_attribute<int32_t>("axis", 0);
+    auto data_shape = datas[0].get_partial_shape();
+    auto data_type = datas[0].get_element_type();
+    OutputVector node_datas_reshape;
+
+    auto axis_const = std::make_shared<opset8::Constant>(element::i64, Shape{}, axis);
+    for (const auto& data : datas) {
+        if (data.get_partial_shape().rank().is_static())
```

Comment:
@luo-cheng2021 do you also think we need this check here? Can not concat support broadcast?

Reply:
I think can be removed here

Reply:
Hi, @ceciliapeng2011 and @luo-cheng2021 :  in paddle [spec of 'stack' op](https://www.paddlepaddle.org.cn/documentation/docs/en/1.7/api/layers/stack.html#stack), we can find that " If x is a list, **the shapes of all these Tensors must be the same.** ", so do you think we'd better keep this check to align with its spec?

Reply:
@liubo-intel  Paddle's check have already guaranteed that this negative case won't happen. That means, no such paddle model can be generated. You can have a try. 

On the other hand, does openvino spec of 'concat' have this limitation as well? If not, we can ignore this check. Even if it is, can we leave it to other modules to throw this kind of exception? We cannot guarantee the case of dynamic shape, anyway.

Reply:
@mvafin  @ilyachur @zhangYiIntel  look forward to your opinion on this check. thanks.

Reply:
> @liubo-intel Paddle's check have already guaranteed that this negative case won't happen. That means, no such paddle model can be generated. You can have a try.
> 
> On the other hand, does openvino spec of 'concat' have this limitation as well? If not, we can ignore this check. Even if it is, can we leave it to other modules to throw this kind of exception? We cannot guarantee the case of dynamic shape, anyway.

agree that, if we can guarantee that no such negative paddle model case,  this check can be removed.

Reply:
I agree that this check is not needed. All required checks will be done when ngraph op is created.

Reply:
have removed this check.